firefox_puppeteer/tests/test_places.py

```diff
@@ -0,0 +1,80 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from firefox_ui_harness.testcase import FirefoxTestCase
+
+
+class TestPlaces(FirefoxTestCase):
+
+    def setUp(self):
+        FirefoxTestCase.setUp(self)
+
+        self.urls = [self.marionette.absolute_url('layout/mozilla_governance.html'),
+                     self.marionette.absolute_url('layout/mozilla_grants.html'),
+                     ]
+
+    def tearDown(self):
+        try:
+            self.places.restore_default_bookmarks()
+            self.places.remove_all_history()
+        finally:
+            FirefoxTestCase.tearDown(self)
+
+    def get_all_urls_in_history(self):
```

Comment:
None of our existing mozmill tests are using that, so I would keep it here for now if that is ok. If needed we could move this later on.